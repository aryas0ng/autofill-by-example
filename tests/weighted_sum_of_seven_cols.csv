1,9,0,2,1,2,3,65
2,8,5,1,16,6,3,174
2,7,4,0,3,1,34,287
3,6,3,2,7,8,4,143
10,5,2,1,8,6,4,134
5,4,1,0,7,1,6,99
4,3,0,9,1,21,4,205
19,2,2,8,8,3,1,126
2,1,1,7,4,4,14,
1,0,0,6,1,6,5,
9,4,6,5,9,15,1,
7,3,5,4,8,3,1,
4,2,4,3,2,4,1,
3,1,3,2,6,1,1,
12,0,2,1,13,1,11,
4,1,1,0,1,9,12,
1,0,0,2,7,4,16,
26,5,3,1,10,2,16,
8,4,2,0,2,2,17,
3,3,1,3,3,3,2,
3,2,0,2,1,7,2,
8,1,4,1,1,23,2,
4,0,3,0,1,3,20,
6,9,2,5,1,8,20,
5,8,1,4,1,3,27,
4,7,0,2,17,6,4,
3,6,6,1,2,7,4,
2,5,5,0,21,3,5,
1,4,4,2,26,1,54,
0,3,3,1,4,8,6,
3,2,2,3,7,6,6,
2,1,1,3,8,4,7,
1,0,0,4,4,4,8,
0,4,3,46,7,17,8,
6,3,2,5,9,5,8,
5,2,1,6,5,11,15,