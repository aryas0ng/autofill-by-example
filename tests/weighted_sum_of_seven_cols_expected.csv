1,9,0,2,1,2,3,65
2,8,5,1,16,6,3,174
2,7,4,0,3,1,34,287
3,6,3,2,7,8,4,143
10,5,2,1,8,6,4,134
5,4,1,0,7,1,6,99
4,3,0,9,1,21,4,205
19,2,2,8,8,3,1,126
2,1,1,7,4,4,14,177
1,0,0,6,1,6,5,101
9,4,6,5,9,15,1,197
7,3,5,4,8,3,1,109
4,2,4,3,2,4,1,73
3,1,3,2,6,1,1,65
12,0,2,1,13,1,11,170
4,1,1,0,1,9,12,152
1,0,0,2,7,4,16,180
26,5,3,1,10,2,16,223
8,4,2,0,2,2,17,163
3,3,1,3,3,3,2,71
3,2,0,2,1,7,2,76
8,1,4,1,1,23,2,183
4,0,3,0,1,3,20,176
6,9,2,5,1,8,20,243
5,8,1,4,1,3,27,252
4,7,0,2,17,6,4,175
3,6,6,1,2,7,4,117
2,5,5,0,21,3,5,185
1,4,4,2,26,1,54,543
0,3,3,1,4,8,6,129
3,2,2,3,7,6,6,138
2,1,1,3,8,4,7,132
1,0,0,4,4,4,8,117
0,4,3,46,7,17,8,394
6,3,2,5,9,5,8,169
5,2,1,6,5,11,15,232